DeUrgenta/Features/CreateGroup/ViewModel/CreateGroupViewModel.swift

```diff
@@ -0,0 +1,34 @@
+import Contacts
+import Foundation
+
+class CreateGroupViewModel: ObservableObject {
+    @Published var contacts = [ContactInfo]()
+
+    var currentContact: ContactInfo?
+    @Published var invitedContacts: [ContactInfo] = []
+
+    init() {}
+
+    func fetchingContacts() {
+        let keys = [CNContactGivenNameKey, CNContactFamilyNameKey, CNContactPhoneNumbersKey]
+        let request = CNContactFetchRequest(keysToFetch: keys as [CNKeyDescriptor])
+        do {
+            try CNContactStore().enumerateContacts(with: request, usingBlock: { contact, _ in
+                self.contacts.append(ContactInfo(firstName: contact.givenName, lastName: contact.familyName, phoneNumber: contact.phoneNumbers.first?.value))
```

Comment:
We should filter out contacts without a phone number since our users won't be able to contact them without it. And in this way, ContactInfo phone number is not optional.